app/main.py

```diff
@@ -54,6 +54,17 @@ async def start_prometheus_metrics():
     ).expose(app)
 
 
+logger = logging.getLogger(__name__ + ".global_taxonomy_refresh")
+
+
+@app.on_event("startup")
+@repeat_every(seconds=60 * 60, logger=logger, wait_first=True)
```

Comment:
I learn today that we only refresh the server page every night, so we could remove cache only every 3h, that seems reasonable.

```suggestion
@repeat_every(seconds=3 * 60 * 60, logger=logger, wait_first=True)
```